tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogService.java

```diff
@@ -169,7 +169,7 @@ PartitionDescProto getPartition(String databaseName, String tableName, String pa
   void addPartitions(String databaseName, String tableName, List<PartitionDescProto> partitions
     , boolean ifNotExists) throws UndefinedTableException, DuplicatePartitionException, UndefinedPartitionMethodException, UndefinedDatabaseException;
 
-  boolean createIndex(IndexDesc index);
+  void createIndex(IndexDesc index) throws DuplicateIndexException;
```

Comment:
If index creation is based on table, it may need ``Undefined(Table|Database)Exception``.